insights_engine/training/data_preprocessing.py

```diff
@@ -106,14 +106,47 @@ def parse_godeps(mercator_op):  # pragma: no cover
     return list(packages_set)
 
 
+def eliminate_duplicates(manifests_all):
+    """Eliminate duplicates in list of all manifests."""
+    # Elimiate duplicates
+    manifest_list_new = []
+
+    for idx, manifest in enumerate(manifests_all):
+        manifest = set(manifest)
+        duplicate = False
+        for man_c in manifest_list_new:
+            if manifest == man_c:
+                duplicate = True
+                break
+        if not duplicate:
+            manifest_list_new.append(manifest)
+    print(len(manifest_list_new))
```

Comment:
Leftover from debugging?

Reply:
Looks to be the case @msrb I've filed an issue: https://github.com/fabric8-analytics/f8a-golang-insights/issues/16 and I'll change all of these to logging statements.